tests/ci/bugfix_validate_check.py

```diff
@@ -36,6 +36,18 @@ def process_result(file_path: Path) -> Tuple[bool, TestResults]:
     test_results = []  # type: TestResults
     state, report_url, description = post_commit_status_from_file(file_path)
     prefix = file_path.parent.name
+    if description.strip() in [
```

Comment:
let's make bugfix check not "always green" again? 
It gets broken and nobody notices, but recently was fixed in https://github.com/ClickHouse/ClickHouse/pull/56383.

Reply:
Actually, I realized that it's not always green (e.g., it failed at https://github.com/ClickHouse/ClickHouse/commit/fd373024f52ccbc9006dcf45bd38f6386243e7ad as it should be). The problem is that if the test failed because of a check failure itself, it would be green. So, I decided to filter such cases with this check, and it will be red in the case of any of these errors